web/v1/redshift/sql-runner/configs/datamodeling.json

```diff
@@ -0,0 +1,45 @@
+{
+  "schema": "iglu:com.snowplowanalytics.datamodeling/config/jsonschema/1-0-0",
+  "data": {
+    "enabled": true,
+    "storage": "Redshift",
+    "sqlRunner": "0.9.2",
```

Comment:
Could it change to  `"storage": "Default"`? (includes Redshift and Snowflake)
Also,since Redshift does not require a different SQL Runner version, would it make sense to remove the `sqlRunner` property?
 (same for: `example_with_custom.json`,`pre_test.json`, `post_test.json`,`teardown_all.json` configs.)

Reply:
Both fair I think. Will change

Reply:
I agree we should change so that the autogen script works out of the box. But @adatzer this raises a question for you. 'Redshift' and 'Snowflake' are IMO the more intuitive values to have in the config. Perhaps downstream we should accept any of them? Food for thought.